crossplane.yaml

```diff
@@ -11,9 +11,11 @@ spec:
   crossplane:
     version: ">=v1.13.2-0"
   dependsOn:
+    - provider: xpkg.upbound.io/upbound/provider-family-azure
```

Comment:
why we need this explicit ?

Reply:
Because it installs the latest/`v0.38.1` as of now and eventually conflicts :)

Reply:
but this is a bug in crossplane package manager or ? why the provider dep is installing `v0.38.1` ?

Reply:
@haarchri this is known behavior of the package manager, I will dig up the discussion

Reply:
@haarchri please see https://github.com/upbound/docs/pull/236/files